gradle.properties

```
@@ -1,7 +1,7 @@
 org.gradle.caching=true
 group=org.ballerinalang
 version=2.1.0-SNAPSHOT
-ballerinaLangVersion=2201.12.0
+ballerinaLangVersion=2201.12.0-20250314-012500-2896b17b
```

I wonder why the lang version is not bumped to 2201.12.1-<timestampe> as we already have the U12 stable release
```suggestion
ballerinaLangVersion=2201.12.0-20250314-012500-2896b17b
```

---

They may have not bumped the version in the lang. The timestamp is correct